src/templates/partials/custom-buttons.html

```diff
@@ -1,5 +1,5 @@
 <div class="custom-buttons">
-    <ul class="social-likes" data-url="http://jqapi.ru" data-title="jQuery API Cheatsheet">
+    <ul class="social-likes" data-url="https://futurecolors.github.io/jqapi.ru/" data-title="jQuery API Cheat Sheet">
```

Comment:
URL 301 redirects to http://source.futurecolors.ru/jqapi.ru/, probably should change it here?